src/languageserver/handlers/settingsHandlers.ts

```diff
@@ -146,6 +153,17 @@ export class SettingsHandler {
     }
   }
 
+  public async updateTabSize(tabSize: number): Promise<boolean> {
```

Comment:
Not sure, why we need to update user settings?

Reply:
As per the bug, when user changed the settings Editor: Detect Indentation is false, then we need to change the default tab size to Editor: Tab size settings value. For that we need to update the user settings from LS.

Reply:
I still not understand why we need to update property. If Editor: Detect Indentation is false formatter should use Editor: Tab, but we don't need to change any property for that

Reply:
Removed the update property code.